portfolio/src/main/webapp/comment-form.js

```diff
@@ -22,19 +22,62 @@ async function postFeed(quantity) {
   formatComments(feed, comments);
 }
 
-/** Format list of user comments such that each is represented as a paragraph */
+/** Format list of user comments such that each is represented as a section */
 function formatComments(container, comments) {
   comments.forEach((comment) => {
-    const pElement = document.createElement('p');
-    pElement.innerText = comment; 
-    pElement.className = 'user-comment';
-    container.appendChild(pElement);
+    const secElement = document.createElement('section');
+    secElement.appendChild(addComment(comment));
+    secElement.appendChild(addLoveFeature(comment));  
+    container.appendChild(secElement);
   });
 }
 
+/** Return a paragraph element that contains a user comment */
+function addComment(comment) {
+  const pElement = document.createElement('p');
+  console.log(comment);
+  pElement.innerText = comment.text; 
+  pElement.className = 'user-comment';
+  return pElement; 
+}
+
+/** Return a div element that contains a love icon and love count */
+function addLoveFeature(comment) {
+  const divElement = document.createElement('div');
+  divElement.className = 'container love-button-container';
+  divElement.appendChild(getLoveIcon(comment));
+  divElement.appendChild(getLoveCount(comment));
+  return divElement; 
+}
+
+/** Return an icon element that contains a love icon */
+function getLoveIcon(comment) {
+  const loveIcon = document.createElement('i'); 
+  loveIcon.className = 'love-button far fa-heart';
+  loveIcon.addEventListener('click', () => {
+    updateLike(comment);
+  });
+  return loveIcon;
+}
+
+/** Return a paragraph element that contains the amount of likes a comment has recieved */
+function getLoveCount(comment) {
+  // TO DO--determine using Datastore how many likes to diplay--0 is placeholder for now
+  const counter = document.createElement('p');
+  counter.innerText = comment.count;
+  return counter; 
+}
+
 /** Delete every comment from the feed */
 async function deleteComments() {
-  // The fetch will delete every comment and return an empty string
+  // response is not meaningful and is just used to catch the fetch return
```

Comment:
You don't need to set the await result to anything. You can do:
await fetch();
...rest of function
And the rest of the function won't run until the await has completed.

Reply:
oh that's much better--adjusted